### PR TITLE
Feat/edit communitites

### PR DESCRIPTION
**Pull request by Jonathansoufer**

### CODE REVIEW

File: src/services/celoWallet.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ async function celoWalletRequest(
         return toTxResult(kit.web3.eth.sendSignedTransaction(tx)).waitReceipt();
     } catch (e) {
         // as transaction requests get pending, they then resume all at once
-        if (e.toLowerCase().includes('known transaction')) {
+        if (e.toLowerCase().includes('failing transaction')) {
```
Comment (obernardovieira) [inline src/services/celoWallet.ts:45]: did you find a "failing transaction"? It usually is a "known transaction".

Comment (Jonathansoufer) [reply src/services/celoWallet.ts:45]: The error was showing failing instead of known when I was testing.

Comment (obernardovieira) [reply src/services/celoWallet.ts:45]: :thinking:

Comment (obernardovieira) [reply src/services/celoWallet.ts:45]: ahh, wait, you mean, that one that says "or always failing transaction"? No, it's a different one. Something like `"message":"known transaction: 84f89c992486ebe6e1258b12b0bbaf8ba604b88ecf0a98acc0989fa3081e8d0d"`

Comment (Jonathansoufer) [reply src/services/celoWallet.ts:45]: ok, I'll rollback this text change.

**Comment by obernardovieira** [comment]
Should we close this without merge?

**Pull request closed without merging**